charts/cluster-api-provider-aws/templates/crd-awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.14.0
+    service.beta.openshift.io/inject-cabundle: "true"
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-aws
+    cluster.x-k8s.io/v1alpha3: v1alpha3
+    cluster.x-k8s.io/v1alpha4: v1alpha4
+    cluster.x-k8s.io/v1beta1: v1beta1_v1beta2
+    clusterctl.cluster.x-k8s.io/move-hierarchy: ""
+  name: awsclustercontrolleridentities.infrastructure.cluster.x-k8s.io
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        caBundle: Cg==
```

Comment:
caBundle causing service error need to be removed. There are 10 CRDs with webhook patch having this caBundle [here](https://github.com/openshift/cluster-api-provider-aws/tree/master/config/crd/patches)

Reply:
the caBundle was removed.